`go mod download` fails because of toml module dependency

_The more information we have the easier it is for us to help. Feel free to remove any sections that might not apply_

### Issue
_Describe what happened and what you were trying to do_

I opened a PR for this repo and CI failed due to an issue with a dependency: https://github.com/stripe/stripe-cli/runs/2440815212?check_suite_focus=true. I believe this means `make setup` will fail on fresh clones of the repo, but I haven't tested this. Maybe we should use a different toml parser?

This is possibly related to https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/

### Expected Behavior
_Tell us what you expected to happen_

This CI step should complete successfully. Locally, `make setup` should complete successfully.

### Steps to reproduce
_What are the steps we can take to reproduce this and verify it's fixed?_

Locally, clean your module cache:
```shell
go clean -modcache
```

Then run `make setup` or `go mod download`:
```shell
❯ make setup                                     
go mod download
go: github.com/BurntSushi/toml@v0.3.1: reading https://gocenter.io/github.com/%21burnt%21sushi/toml/@v/v0.3.1.mod: 405 Not Allowed
make: *** [setup] Error 1
```

### Traceback
_Share any debug output that was given by the CLI_

### Environment
_Select one of: macOS, Linux, Windows, Other_

Locally, macOS.
In GH Actions, Linux.

--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,6 @@ TEST_OPTIONS?=
 export GO111MODULE := on
 export GOBIN := $(shell pwd)/bin
 export PATH := $(GOBIN):$(PATH)
-export GOPROXY := https://gocenter.io
 export GOLANGCI_LINT_VERSION := v1.37.1
 
 # Install all the build and lint dependencies

